airbyte-server/src/main/java/io/airbyte/server/handlers/DestinationDefinitionsHandler.java

```diff
@@ -34,6 +34,8 @@
 
 public class DestinationDefinitionsHandler {
 
+  private static final String DEV_TAG = "dev";
```

Comment:
worth creating a `SeverConstants` class and putting this in there so it can be shared across `DestinationDefinitionsHandler` and `SourceDefinitionsHandler`?

Reply:
Done